apps/web/pages/settings/organizations/[id]/onboard-admins.tsx

```diff
@@ -0,0 +1,41 @@
+import Head from "next/head";
+import { useRouter } from "next/router";
+
+import { AddNewOrgAdminsForm } from "@calcom/features/ee/organizations/components";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { WizardLayout } from "@calcom/ui";
+
+import PageWrapper from "@components/PageWrapper";
+
+const OnboardTeamMembersPage = () => {
+  const { t } = useLocale();
+  const router = useRouter();
+  if (!router.isReady) return null;
+  return (
+    <>
+      <Head>
+        <title>Invite your organization admins</title>
+        <meta
+          name="description"
+          content="These admins will have access to all teams in your organization. You can add team admins and members later."
+        />
```

Comment:
same as above

Reply:
Resolved in the [correct PR](https://github.com/calcom/cal.com/pull/9184) for these changes, thanks!